src/program.cpp

```diff
@@ -248,14 +277,18 @@ std::string GetEffectiveLine(std::fstream& file, int& lineNumber, int& lineStart
         if(LineIsWhitespace(newLine))
             continue;
 
+        if (LineIsComment(newLine))
+            continue;
+
         if(FirstTime)
         {
             lineStart = lineNumber - 1;
             FirstTime = false;
         }
         fullLine += newLine;
 
-    } while (LineIsWhitespace(newLine) || LastNonWhitespaceChar(newLine) == ',');
+    } while (LineIsWhitespace(newLine) || LastNonWhitespaceChar(newLine) == ',' 
```

Comment:
So `LastNonWhitespaceChar` being `,` is not really a thing anymore... you can just get rid of it in this feature no one will miss it :)

Reply:
I did remove this, but we could add it back if we wanted to support multiple lines.